wordpress-mobile/WordPress-iOS
# Comment Moderation Bar: restrict width in landscape

Ref: #17200

This change does a few things:
- Adds the moderation bar back to the `CommentContentTableViewCell`. It is shown if the user has moderation rights.
- Restricts the button view width in iPad landscape and regular iPhone landscape.
  - The gray background still extends the full width.
  - The button stack view horizontal margins are increased so the buttons are not severely stretched out.

NOTE: I'll get Miklic's input when the bar UI is finalized to reduce the number of times he has to review it.

To test:

---
- Enable the `newCommentDetail` feature.
- On a site where you have permission to moderate comments, go to My Site > Comments > Comment details.
  - Verify the moderation bar is shown.
- On a site where you do not have permission to moderate comments, go to My Site > Comments > Comment details.
  - Verify the moderation bar is not shown.

| Shown | Hidden |
|--------|-------|
| <img width="470" alt="Screen Shot 2021-09-28 at 3 18 49 PM" src="https://user-images.githubusercontent.com/1816888/135167270-d93a7576-99b0-49af-ab32-0c081cac7226.png"> | <img width="471" alt="Screen Shot 2021-09-28 at 3 20 48 PM" src="https://user-images.githubusercontent.com/1816888/135167398-fb4afa20-dbf4-4525-aff2-9bd80e328669.png"> |

---
- Run on an iPad, and iPhone without split view.
- Rotate the device.
- Verify the buttons view width is restricted.

| iPad | iPhone |
|--------|-------|
| <img width="449" alt="Screen Shot 2021-09-28 at 3 13 08 PM" src="https://user-images.githubusercontent.com/1816888/135166640-b5150a49-3a83-4881-9202-258dad87bcc4.png"> | <img width="481" alt="Screen Shot 2021-09-28 at 4 13 45 PM" src="https://user-images.githubusercontent.com/1816888/135173320-1918621f-3ffc-46ff-a586-556dd883ed8e.png"> |
| <img width="932" alt="Screen Shot 2021-09-28 at 3 13 25 PM" src="https://user-images.githubusercontent.com/1816888/135166649-866f3e3d-5a93-4ecb-b7a0-64fa795757d7.png"> | <img width="901" alt="Screen Shot 2021-09-28 at 4 14 21 PM" src="https://user-images.githubusercontent.com/1816888/135173330-81af98eb-8d43-474e-ad40-12abc7d61c87.png"> |

---
## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete and disabled.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete and disabled.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete and disabled.

PR submission checklist:
